ci-operator/step-registry/cucushift/installer/rehearse/gcp/ipi/xpn/minimal-permission/byo-zone/cucushift-installer-rehearse-gcp-ipi-xpn-minimal-permission-byo-zone-workflow.metadata.json

```diff
@@ -0,0 +1,15 @@
+{
+	"path": "cucushift/installer/rehearse/gcp/ipi/xpn/minimal-permission/byo-zone/cucushift-installer-rehearse-gcp-ipi-xpn-minimal-permission-byo-zone-workflow.yaml",
```

Comment:
```suggestion
	"path": "cucushift/installer/rehearse/gcp/ipi/xpn/minimal-permission/byo-hosted-zone/cucushift-installer-rehearse-gcp-ipi-xpn-minimal-permission-byo-hosted-zone-workflow.yaml",
```

Reply:
OK, let me update after the current test done. Thanks!